test/fixtures/dialects/postgres/postgres_alter_database.sql

```diff
@@ -0,0 +1,29 @@
+ALTER DATABASE db;
+ALTER DATABASE db ALLOW_CONNECTIONS true;
+ALTER DATABASE db WITH ALLOW_CONNECTIONS true;
+ALTER DATABASE db CONNECTION LIMIT 10;
+ALTER DATABASE db WITH CONNECTION LIMIT 10;
+ALTER DATABASE db IS_TEMPLATE true;
+ALTER DATABASE db WITH IS_TEMPLATE true;
+ALTER DATABASE db IS_TEMPLATE true ALLOW_CONNECTIONS true;
+ALTER DATABASE db WITH IS_TEMPLATE true ALLOW_CONNECTIONS true;
```

Comment:
Maybe a test case with ALLOW_CONNECTIONS, CONNECTION LIMIT, and IS_TEMPLATE all specified together?